src/ui/modifier-select-ui-handler.ts

```diff
@@ -180,6 +202,11 @@ export default class ModifierSelectUiHandler extends AwaitableUiHandler {
       this.options.push(option);
     }
 
+    // Set "Continue" button height based on number of rows in healing items shop
+    const continueButton = this.continueButtonContainer.getAt(0) as Phaser.GameObjects.Text;
```

Comment:
NIT:
`getAt()` support generic types
```suggestion
    const continueButton = this.continueButtonContainer.getAt<Phaser.GameObjects.Text>(0);
```